R/03-geojob-obj.R

```diff
@@ -56,6 +56,31 @@ setMethod("geojob", signature(), function(...) {
   return(geojob)
 })
 
+#'@param xml location of xml 
+#'@rdname geojob-methods
+#'@importFrom httr GET 
+#'@importFrom RCurl url.exists
+#'@importFrom XML xmlTreeParse xmlAttrs xmlRoot
+#'@aliases geojob,geojob-method
+setMethod("geojob", signature("ANY"), function(xml, urlSlot = NULL) {
+  ## create new geojob object
+  job <- new("geojob")
+  #parse based on xml class
+  if(is.character(xml)) {
+    if(url.exists(xml)){
+      doc <- GET(xmlLocation)
```

Comment:
for `GET`s in this package, use the function in the package `gGET`, which does some other things on top of `httr::GET` such as do a retry and add the `user-agent` string to the request.